apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -4,7 +4,7 @@ import JavalabDropdown from './components/JavalabDropdown';
 import javalabMsg from '@cdo/javalab/locale';
```

Comment:
If you get the chance, this would be a good opportunity to also add reactOnClickOutside so the dropdown auto-closes when clicking out of it (similar to JavaLabFileExplorer)